Adding a new reminder

becs919 opened the pull request:
@martensonbj @Tman22 (if you have any advice on the date problem, what would be great!)
When user is on root, they click a create button and it takes them to reminders/new
Added to list of reminders as you type the title
After clicking submit, you can see the body but NOT THE DATE??
closes partially issue #4 

## Purpose
We allowed the user to add a reminder (with exception of date!) and have it render on the page!  

## Approach
We first made a button that takes the user to the reminders/new page. Then with the help of Taylor, got our inputs and actions together to save a reminder and render on the page. 

### Learning
Lot's of learning for this one! Thanks Brenna for the docs you send over, also Taylor for all of your help! 

### Open Questions and Pre-Merge TODOs
-Need to figure out why the date isn't capturing 
-Need to clear input fields on click 

### Test coverage 
Added two acceptance tests for new. Passing! 

### Follow-up tasks
Finish getting date to capture as well as clearing input fields on click

and issue #5 

### Screenshots
![after](http://g.recordit.co/RrhKsJJeH7.gif)

becs919 committed:
Adding a new reminder

When user is on root, they click a create button and it takes them to reminders/new
Added to list of reminders as you type the title
After clicking submit, you can see the body but NOT THE DATE??

Tman22 commented on app/templates/reminders/new.hbs:
```diff
@@ -0,0 +1,7 @@
+<div class='add-new-reminder'>
+  {{input type='text' value=model.title placeholder='Enter a title'}}
+  {{log model.title}}
```
Take this out

Tman22 commented on tests/acceptance/new-test.js:
```diff
@@ -0,0 +1,43 @@
+  server.createList('reminder', 5);
+
+  visit('reminders/new')
+
```
Can you fix the spacing to be consistent here.

Tman22 commented on tests/acceptance/new-test.js:
```diff
@@ -0,0 +1,43 @@
+test('adding a new reminder', function(assert) {
+  server.createList('reminder', 5);
+
+  visit('reminders/new')
```
Semi-colons?

becs919 committed:
Fixed spacing, semi colons, and deleted the log

Per Taylor's request

becs919 commented:
@Tman22 @martensonbj 
Deleted semi colons, spacing, and log per Taylor's request. We still aren't rendering the date though for this issue!

becs919 committed:
Add new reminder - got date to save

Changed DS.attr('date') it 'string'
Now date saves when user clicks submit

becs919 commented:
@Tman22 @martensonbj 
We got the date to save and render on the page with this last commit. Now we only have to figure out how to clear the fields after submit.

martensonbj commented:
Nice work guys! in terms of clearing the fields, think about your controller/component as the "internal state" of your template. Check out this example from [ember-groceries](https://github.com/turingschool-examples/ember-groceries/blob/master/app/components/add-grocery.js) - notice in particular how after the record is created you call `setProperties()` and reset the properties you are targeting to empty strings.  

@becs919

becs919 committed:
Finish add new reminder

changed from model.'input' to a variable
in controller we are:
  -getting values
  -saving reminder
  -clearing inputs

becs919 commented:
@Tman22 @martensonbj 

We finished issue 4 and can close #4 

Check it out! ![](http://g.recordit.co/Oc7NnN5vhl.gif)

martensonbj commented:
Good work team! Way to push through.
